### PR TITLE
CAPSTONE PROJECT MODULE ONE

### PR DESCRIPTION
**Pull request by HanningtoneM**
- Used semantic HTML tags.
- Applied best practices in HTML code.
- Used CSS selectors correctly.
- Used CSS box model.
- Used Flexbox to place elements in the page.
- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
- Used GitHub Pages to deploy web pages.
- Applied JavaScript best practices and language style guides in code.
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events.
- Used objects to store and access data.
- Communicated technical concepts to other technical people.
- Chose a topic that is different than the one in the original design.

- used a CSS framework (Bootstrap) for styling
- built these 2 pages: The home page. The about page.

- https://www.loom.com/share/d6c51b61b1f04562b520058f1ebf1a85 part one of the loom video
- https://www.loom.com/share/ec06a4d695094c7ab11ad55e876e88c8 part two of the loom video

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+  <h2 class="text-center seconday-dark">Kenya Women’s Summit Logo</h2>
+  <div class="line"></div>
+  <div class="logoDescripiton">
+    <p class="text-center desktopDescription">
+      The Kenya women's summit brings together high-achieving women from all over the country. 
+      It gives focus to new ways of thinking and presents aspects we think we already understand from a new and unfamiliar perspective. 
+      This summit enables women to start a dialogue; women who shape their societies and their organisations – women who make their mark.
+      It directs attention to new markets and fields of action for companies and regions, enables the joint exploration of visions and strategies, 
+      and extends the boundaries of previous knowledge. 
+    </p>
```
Comment (keroloussamy) [inline about.html]: - [x] Kindly the text here overlaps each other, and maybe it looks good on your screen, but you have also to be sure that your design works correctly on all widths.
- [x] Kindly you can give the paragraph some margin from left and right to match the original design.

Your design
image

Original design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+          <img src="./images/partner/microsoft.svg" alt="microsoft">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/zoom-communications.svg" alt="zoom">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/airbnb.svg" alt="airbnb">
+      </div>
+  </div>
+</article>
```
Comment (keroloussamy) [inline about.html]: - [x] Kindly this section doesn't exist on the about page, it shows on the mobile version but on the desktop version, it does not exist.

Your design
image

Original design
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+      </div>
+      <div class="footerInfo">
+        <h4>@2022 Women's Summit.</h4>
+        <p class="mt-3">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Omnis,
+          suscipit veritatis!?
+        </p>
+      </div>
+    </div>
+  </footer>
```
Comment (keroloussamy) [inline about.html]: - [x] Kindly change the background color for the footer section on the about page, to match the original design.

Your design
image

Original design
image

File: README.md [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+
+👤 **hanningtone Monda**
+
+- Github: [@HanningtoneM](https://github.com/HanningtoneM)
+
+
+## Credits
+
+Template from this [Figma](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Microverse](https://bit.ly/MicroverseTN)
+
```
Comment (keroloussamy) [inline README.md]: - [x] Kindly you can mention the name of the author [Cindy Shin](https://www.behance.net/adagio07) instead of Microverse.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @HanningtoneM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HanningtoneM** [comment]
fixed issues requested by the reviewer

- mentioned the name of the author [Cindy Shin](https://www.behance.net/adagio07) 
- I changed the background color for the footer section on the about page, to match the original design.
- I removed the partners section on the about page
- fixed text overlapping issue
- centered paragraph to appear like the original design

File: README.md [hunk 5]
```diff
@@ -1,2 +1,46 @@
 # Women-summit
 women's summit to highlight some of the problems facing women
+
```
Comment (rustam-tolipov) [inline README.md:3]: - [x] [OPTIONAL] Good job 👏, all requirements from the previous review are implemented ✔. I would suggest adding a screenshot for the website. Let's also add Twitter and LinkedIn account links in the **author** part 🙌

**Review by rustam-tolipov — APPROVED** [review]
## Hi @HanningtoneM 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**